test/matrix.lean

```diff
@@ -1,7 +1,3 @@
-/-
-manually ported from
-https://github.com/leanprover-community/mathlib/blob/4f4a1c875d0baa92ab5d92f3fb1bb258ad9f3e5b/test/matrix.lean
--/
```

Comment:
Please stop deleting this line every time you edit this file!

Reply:
Oh, sorry, it is on `nightly-testing` so keeps getting incorporated back into bump PRs. I'll clean up in both locations now.

Reply:
Done.

Reply:
Thanks!